theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,25 @@
-<nav>
-  <ul>
-    <% for (const {title, link} of site.navigation) {%>
-      <li>
-        <a href="<%- link %>" target="_blank"><%- title %></a>
-      </li>
-    <% } %>
-  </ul>
+<select id="mobile-nav">
+      <% for ({title, link, sub_items} of site.navigation) { %>
```

Comment:
Detail/suggestion: Unfortunately there wasn't a linter and code formatter that would support EJS files, so for now let's try keeping the file consistent as much as we can, for example here we can introduce 2 level indentation instead of 6, to make the code more compact and consistent, same applies to other inconsistent identations in the file . It should be easy to setup identation in the VSCode -> https://stackoverflow.com/a/38556923/1439474, so when you hit TAB it will use "2 spaces".